controllers/kafkatopic_controller.go

```diff
@@ -32,10 +35,18 @@ import (
 	"github.com/banzaicloud/koperator/pkg/k8sutil"
 	"github.com/banzaicloud/koperator/pkg/kafkaclient"
 	"github.com/banzaicloud/koperator/pkg/util"
+	"github.com/banzaicloud/koperator/pkg/webhooks"
 )
 
 var topicFinalizer = "finalizer.kafkatopics.kafka.banzaicloud.io"
 
+func isTopicManagedByKoperator(topic metav1.Object) bool {
+	if managedByAnnotation, ok := topic.GetAnnotations()[webhooks.TopicManagedByAnnotationKey]; !ok {
```

Comment:
I believe it is `ok` instead of `!ok`? I got a bit lost following the current logic: 

- when the "managedBy" annotation doesn't exist, we are returning `"" == webhooks.TopicManagedByKoperatorAnnotationValue`, which is always false
- and when "managedBy" annotation does exist, we are hitting the "return false" statement at the end of this function

It would be great to add a unit test for this little function

Reply:
Nice catch!
Actually when I changed based Pregnor suggestion. It went wrong :) (https://github.com/banzaicloud/koperator/pull/934#discussion_r1112859931)

Reply:
I added test